components/KiboAppHeader.vue

```diff
@@ -18,11 +18,38 @@
         <div style="flex: 0.3"></div>
         <div class="kibo-header__search-bar">
           <SfSearchBar
-            :value="searchValue"
-            placeholder="Search for items"
+            ref="searchBarRef"
+            v-click-outside="closeSearch"
+            :placeholder="$t('Search for items')"
             aria-label="Search"
             class="sf-header__search"
-          />
+            :value="term"
+            @input="handleSearch"
+            @keydown.enter="handleSearch($event)"
```

Comment:
On enter we want to clear the searchbar input, close the Search Suggestions and redirect to the /search page

Reply:
resolved with icky-305